giraffe/src/utils/fromFlux.ts

```diff
@@ -277,6 +277,13 @@ const parseValue = (value: string | undefined, columnType: ColumnType): any => {
   }
 
   if (columnType === 'number') {
+    if (value === '+Inf') {
+      return '+Inf'
+    }
+
+    if (value === '-Inf') {
+      return '-Inf'
+    }
```

Comment:
This solution won't last forever. What if there is another value added in the future that can't be forced into a number? I think, ideally, the number column should just gracefully treat non-numbers as the original string value. Something like this:

```
  if (columnType === 'number') {
    const parsedValue = Number(value)
    return parsedValue === parsedValue ? parsedValue : value
  }
```

In the `return` statement, NaN will fail the first condition and the original string will be returned, which will be whatever was in the flux CSV